internal/source/source.go

```diff
@@ -346,6 +353,10 @@ func templateMsg(messageTemplate string, templateValues *TemplateData) (string,
 		messageTemplate = defaultMessageTemplate
 	}
 
+	if deprecatedUpdatedPattern.MatchString(messageTemplate) {
+		return "", ErrDeprecatedTemplateField
+	}
```

Comment:
While discussing this with @stefanprodan it was raised that this kind of check might be too risky. We would prefer to parse the error from `t.Execute()` below. I just ran a test locally here with the field removed, and this is the error I get:

```
"failed to run template from spec: template: commit message:6:34: executing \"commit message\"
at <.Updated.Files>: can't evaluate field Updated in type source.TemplateData"
```

So if we just check if the error string contains the substring:

`can't evaluate field Updated in type source.TemplateData`

This should work without false positives.